docs/blog/2021-01-11-qrl-joins-league-of-entropy.md

```diff
@@ -0,0 +1,24 @@
+---
+title: QRL joins The League of Entropy
+summary: The Quantum Resistant Ledger becomes the latest member of the League of Entropy.
+date: 2021-01-11
+tags:
+    - New members
+    - League of Entropy
+    - News
+---
+
+On Wednesday, December 16, 2020, The League of Entropy, running an 
+[upgraded version of the drand protocol](https://github.com/drand/drand/releases/tag/v1.2.5), 
+added [The Quantum Resistant Ledger (QRL)](https://www.theqrl.org/) to its list of members. 
+QRL describes itself as, "An externally audited enterprise-grade blockchain platform secure 
+today from the quantum computing advances of tomorrow." Its related foundation, 
+[The QRL Foundation](https://qrl.foundation/),  is a not-for-profit organisation supporting 
+the research and development of open-source post-quantum blockchain solutions..
```

Comment:
```suggestion
On Wednesday, December 16, 2020, The League of Entropy, running an 
[upgraded version of the drand protocol](https://github.com/drand/drand/releases/tag/v1.2.5), 
added [The Quantum Resistant Ledger (QRL)](https://www.theqrl.org/) to its list of members. 
QRL describes itself as "An externally audited enterprise-grade blockchain platform secure 
today from the quantum computing advances of tomorrow." Its related foundation, 
[The QRL Foundation](https://qrl.foundation/), is a not-for-profit organisation supporting 
the research and development of open-source post-quantum blockchain solutions.
```

(minor fixes)

Reply:
Is it accurate to say QRL joined or should we say the QRL Foundation joined? I'd like to avoid repeating the EF mistakes and it seems to me that we're dealing with the foundation and not the abstract project. If they prefer it this way, though, it's fine with me.

There is definitely a better phrasing than "Its related foundation" but it depends on the answer to the Q above.

Reply:
Very different from EF, just given the (lack of) size and scope of QRL. Previously from QRL:

> Recommend the QRL project logo (icon + ‘QRL’) and link to theqrl.org — the Foundation website has yet to be refreshed to new branding and is less rich in terms of content, and _given the overlap of project contributors/Foundation volunteers I’m sure this will be fine._

I also ran the blog text by JP and he approved. I went with QRL rather than QRL Foundation to be consistent with the logo and representation on the website.

Reply:
Works for me!